src/localization/nav2_amcl/test/integration/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+include_directories( ${PROJECT_SOURCE_DIR}/src )
```

Comment:
In the future I'd like to split out the ROS-level integration test into a separate directory. It isn't technically dependent on this implementation. OK for this PR.